### PR TITLE
[FEATURE] update request by id - PATCH

### PR DESCRIPTION
**Pull request by iariasleon**
@XavierVal @fgalan

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature [hunk 1]
```diff
@@ -2869,7 +2893,7 @@ Feature: update or append an attribute by entity ID in NGSI v2. "POST" - /v2/ent
       | Fiware-Service     | test_update_op   |
       | Fiware-ServicePath | /test            |
       | Content-Type       | application/json |
-    When update or append an attribute by ID "room"
+    When update or append attributes by ID "room"
       | parameter            | value       |
       | attributes_name      | temperature |
       | attributes_value     | 80          |
```
Comment (manucarrace) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: Consider to split this feature in several features. 2875 lines ... NTC

File: test/acceptance/behave/environment.py [hunk 2]
```diff
@@ -79,6 +82,7 @@ def after_feature(context, feature):
     :param feature: feature properties
     """
     context.execute_steps(u'Given stop service')
+    print colored('    And stop context Broker', 'green')
```
Comment (manucarrace) [inline test/acceptance/behave/environment.py]: print statement. Remove or use logger

Comment (iariasleon) [inline test/acceptance/behave/environment.py]: this print is used to show some steps that are execute manually and it is not a normal `print` use `termcolor` library.

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: This feature, group all scenarios into a request, but in behave, the execution is very fast.

**Comment by manucarrace** [comment]
LGTM

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature [hunk 3]
```diff
@@ -596,21 +604,37 @@ Feature: update or append an attribute by entity ID in NGSI v2. "POST" - /v2/ent
...
+      | temp-34    |
+      | TEMP34     |
+      | house_flat |
+      | house.flat |
+      | house-flat |
+      | house@flat |
+      | habitación |
+      | españa     |
+      | barça      |
+      | random=10  |
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: This is literarly "random=10"? Or a construct that generated ten random rows in the examples table?

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature [hunk 4]
```diff
@@ -1805,23 +1829,23 @@ Feature: update or append an attribute by entity ID in NGSI v2. "POST" - /v2/ent
...
+      | attributes_name | humidity               |
       | metadatas_name  | <attributes_meta_name> |
       | metadatas_value | 5678                   |
     Then verify that receive an "Created" http code
-    And verify that entities are stored in mongo
+    And verify that an entity is updated in mongo
     Examples:
       | attributes_meta_name |
       | 34                   |
-    #  | 34.4E-34             |
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: As the test is now skipeed, maybe the line could be uncomented (same in two more cases, below).

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature (hunk 3 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: In this case, the value is a random generated automatically with length of 10 chars. Characters allowed are [a-zA-Z0-9_]. Ej `DFG435sfSf`

in another cases, ej: random=100000 is imposible to define a value in the feature, therefore it is generated randomly and automatically.

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature (hunk 4 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: this comments were remove previously, see 
https://github.com/telefonicaid/fiware-orion/pull/1301/files#diff-dd39181a1607a58d5f1299e44ed4cd38R1842

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature (hunk 3 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: see https://github.com/telefonicaid/fiware-orion/blame/develop/test/acceptance/behave/README.md#L195

Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: NTC

File: test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature (hunk 4 above)
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**